packages/p2p/src/pages/app.jsx

```diff
@@ -137,7 +137,7 @@ const App = () => {
             p2p_settings.currency_list.forEach(currency => {
                 const { is_default, value } = currency;
 
-                if (is_default && !buy_sell_store.selected_local_currency) {
+                if (is_default) {
```

Comment:
this caused issues with the currency dropdown always being set to the previous local currency, we shouldn't check if it already is set to a value, and allow it to keep changing